sdk/identity/azure-identity/azure/identity/aio/_credentials/azure_powershell.py

```diff
@@ -74,10 +81,13 @@ async def get_token(self, *scopes: str, **kwargs: Any) -> AccessToken:
         """
         # only ProactorEventLoop supports subprocesses on Windows (and it isn't the default loop on Python < 3.8)
         if sys.platform.startswith("win") and not isinstance(asyncio.get_event_loop(), asyncio.ProactorEventLoop):
-            return _SyncCredential().get_token(*scopes, **kwargs)
+            return _SyncCredential().get_token(*scopes, tenant_id=tenant_id, **kwargs)
```

Comment:
Why no claims?

Reply:
I removed it since the synchronous credential doesn't use `claims`. Passing it along wouldn't have an effect, but I cleaned this up as part of removing `claims` uses that don't do anything